make/modules/java.compiler/Java.gmk

```diff
@@ -25,3 +25,6 @@
 
 DOCLINT += -Xdoclint:all/protected \
     '-Xdoclint/package:java.*,javax.*'
+
+EXCLUDES += \
+    javax/tools/snippet-files \
```

Comment:
You can put this just on a single line :-). 

And I'm frankly not sure if make is happy about having a trailing backslash but no additional line...

Reply:
As long as the next line is empty, it works, but it's not a good idea. That's why we came up with the terminating # in our 1 element per line lists. In this case, I would prefer a single line assignment without any backslashes.

Reply:
Fixed to single line